### ISSUE TITLE
Picking segfault on line mesh

### ISSUE THREAD

**Issue by dlyr** [STORM-IRIT/Radium-Engine#285]
Hi,
I haven't tested a lot, but when I add a RM_LINES render object, it raise an OpenGL invalid operation 
```c++
meshBezier->setRenderMode( Engine::Mesh::RM_LINES );
/* load geometry and set config */
    Engine::RenderObject *ro =
        new Engine::RenderObject( "bezierLine", tip, Engine::RenderObjectType::Fancy );
    ro->setMesh( meshBezier );
    Ra::Engine::RadiumEngine::getInstance()->getRenderObjectManager()->addRenderObject( ro );
```
Relevant trace:
```bash
- 12:34:54 ERROR: OpenGL error (/media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Mesh/Mesh.cpp:49, glDrawElements( static_cast<GLenum >(m_renderMode), m_numElements, GL_UNSIGNED_INT, (void*)0 )) :  Invalid operation : The specified operation is not allowed in the current state. The offending command is ignored and has no other side effect than to set the error flag.
(GL_INVALID_OPERATION : 0xGL_INVALID_OPERATION).

Thread 1 "painty" received signal SIGTRAP, Trace/breakpoint trap.
bt
Ra::Engine::Mesh::render (this=0x555555faac60)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Mesh/Mesh.cpp:51
51	        }
(gdb) bt
#0  0x00007ffff70e1374 in Ra::Engine::Mesh::render() (this=0x555555faac60)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Mesh/Mesh.cpp:51
#1  0x00007ffff71fbda7 in Ra::Engine::Renderer::renderForPicking(Ra::Engine::RenderData const&, std::array<Ra::Engine::ShaderProgram const*, 3ul> const&, std::array<std::vector<std::shared_ptr<Ra::Engine::RenderObject>, std::allocator<std::shared_ptr<Ra::Engine::RenderObject> > >, 3ul> const&) (this=0x555557c454c0, renderData=..., pickingShaders=..., renderQueuePicking=...)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Renderer.cpp:346
#2  0x00007ffff71fccd1 in Ra::Engine::Renderer::doPicking(Ra::Engine::RenderData const&) (this=0x555557c454c0, renderData=...)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Renderer.cpp:380
#3  0x00007ffff71faf2a in Ra::Engine::Renderer::render(Ra::Engine::RenderData const&) (this=0x555557c454c0, data=...)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/Engine/Renderer/Renderer.cpp:160
#4  0x00007ffff64622f9 in Ra::Gui::Viewer::startRendering(float) (this=
    0x5555565dfa90, dt=0.0166666675)
    at /media/jupi/Repositories/painty/external/Radium-Engine/src/GuiBase/Viewer/Viewer.cpp:533
```

**Comment by hoshiryu** [comment]
As stated in #269 , picking hasn't been tested on line meshes, especially the line-specific picking shader might be inconsistent.

**Comment by dlyr** [comment]
Ok but where i should add a guard to not call picking rendering on line mesh? Because segfault is a bit annoying and picking is called whenever a I click in the viewer widget (i do not actually want to pick this mesh).

**Comment by hoshiryu** [comment]
Picking is erformed in the Renderer, you can prevent filling the picking render queue for line meshes for now

**Comment by nmellado** [comment]
Temporary fix is proposed in #286.

**Comment by nmellado** [comment]
A proper fix is still waiting for this.
